assets/src/utils/minio_utils.py

```diff
@@ -14,14 +15,21 @@
 
 MinioClient = minio.Minio(
     endpoint=settings.endpoint,
-    access_key=settings.minio_access_key,
-    secret_key=settings.minio_secret_key,
+    access_key=settings.MINIO_ROOT_USER,
+    secret_key=settings.MINIO_ROOT_PASSWORD,
+    secure=False,
```

Comment:
Create config variable `MINIO_SECURE_CONNECTION` and use this value from config

Reply:
done